docs/src/pages/components/links/UnderlineLink.tsx

```diff
@@ -1,27 +1,21 @@
-/* eslint-disable jsx-a11y/click-events-have-key-events */
-/* eslint-disable jsx-a11y/no-static-element-interactions */
 /* eslint-disable jsx-a11y/anchor-is-valid */
 import * as React from 'react';
-import { makeStyles, createStyles, Theme } from '@material-ui/core/styles';
+import { experimentalStyled as styled } from '@material-ui/core/styles';
 import Link from '@material-ui/core/Link';
+import * as CSS from 'csstype';
 
-const useStyles = makeStyles((theme: Theme) =>
-  createStyles({
-    root: {
-      ...theme.typography.body1,
-      '& > * + *': {
-        marginLeft: theme.spacing(2),
-      },
-    },
-  }),
-);
+const Box = styled('div')(({ theme }) => ({
```

Comment:
Why create a custom Box component?

Reply:
For the `...theme.typography.body1`. Who is it with `Box`?

Reply:
https://next.material-ui.com/system/typography/#variant

Reply:
Resolved